lib/providers/env_var_provider.ex

```diff
@@ -209,16 +226,12 @@ defmodule EnvVar.Provider do
     |> Enum.map(&convert(&1, type))
   end
 
-  defp set_default(value, default, _env_var_name) when is_nil(value) do
+  defp set_default(value, default) when is_nil(value) or value == "" do
     default
   end
 
-  defp set_default(value, default, _env_var_name) do
-    if String.length(value) < 1 do
```

Comment:
I think this is equivalent to `value == ""`, right?

Reply:
Hmm, I'm confused about the previous behavior for a variable that is set but has empty string now.